roxie/ccd/ccdserver.cpp

```diff
@@ -21486,7 +21486,7 @@ class CRoxieServerDiskReadActivityFactory : public CRoxieServerActivityFactory
         {
             bool isOpt = (helper->getFlags() & TDRoptional) != 0;
             OwnedRoxieString fileName(helper->getFileName());
-            datafile.setown(_queryFactory.queryPackage().lookupFileName(fileName, isOpt, true, true, _queryFactory.queryWorkUnit()));
+            datafile.setown(_queryFactory.queryPackage().lookupFileName(fileName, isOpt, true, true, _queryFactory.queryWorkUnit(), !queryFactory.isDynamic()));
```

Comment:
For those that aren't currently checking but should.. should we make that change in this issue?